lib/std/math/big/int_test.zig

```diff
@@ -517,8 +515,8 @@ test "big.int add multi-single" {
 }
 
 test "big.int add multi-multi" {
-    var op1: u128 = 0xefefefef7f7f7f7f;
-    var op2: u128 = 0xfefefefe9f9f9f9f;
+    const op1: u128 = 0xefefefef7f7f7f7f;
+    const op2: u128 = 0xfefefefe9f9f9f9f;
```

Comment:
Often `var` is used to ensure that a value is present at runtime.
So this PR has the unintended side effect of breaking a bunch of test cases, that specifically test runtime behavior.
Changing the `var`s here to `const` for example makes the test case tautological, see also #13923.